GeneralPlugin/Android/plugin-manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+    "api": {
+        "require_startup_execution": false,
+        "class_name": "com.applicaster.web.plugins.iai.IAIWebAdapter"
```

Comment:
`iai` is from a specific existing plugin right?
generalize this please

Reply:
I have removed it